src/features/posts/components/add-post-form.tsx

```diff
@@ -64,17 +88,41 @@ export const AddPostForm = () => {
                     id="standard-basic" 
                     label="Title" 
                     variant="standard" 
-                    name="post-title"
+                    name="post-title" 
                     sx={{width: '100%'}}
                     error={error.title ? true : false}
                     helperText={error.title}
                 />
-                <ReactQuill
-                    theme="snow" 
-                    value={content}
+                <Editor
+                    apiKey={ApiKeyTinyMMC}
                     onChange={handleContentChange}
+                    init={{
+                        plugins: 'mentions anchor autolink charmap codesample emoticons image link lists media searchreplace table visualblocks wordcount checklist mediaembed casechange export formatpainter pageembed permanentpen footnotes advtemplate advtable advcode editimage tableofcontents mergetags powerpaste tinymcespellchecker autocorrect a11ychecker typography inlinecss',
+                        toolbar: 'undo redo | blocks fontfamily fontsize | bold italic underline strikethrough | link image media table mergetags | align lineheight | checklist numlist bullist indent outdent | emoticons charmap | removeformat',
+                        width: '800px',
```

Comment:
Do przeniesienia jako nowy komponent, ewentualnie przeniesienie  init do jakiegoś consta, bo tak to bardzo zaśmieca ta lista pluginów, toolbaru, megetags itp